diff.py

```diff
@@ -131,6 +131,15 @@ def fail(msg):
     help="Diff algorithm to use.",
 )
 
+parser.add_argument(
+    "--max-size",
+    "--max-lines",
+    dest="max_lines",
+    type=int,
+    default=1024,
+    help="The maximum length of the diff, in lines. Not recommended when -f is used.",
```

Comment:
```suggestion
    help="The maximum length of the diff, in lines. Not recommended when -s is used.",
```

(or drop the note about -s altogether; currently it *is* actually useful to combine them, since the -s filter is applied after the disassembly limit)